### DISCUSSION TITLE
Get contents/body of issue in action

### DISCUSSION THREAD

**Discussion by thechelsuk** [community/community#25486] [Actions]

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/matbenfield">@MatBenfield</a>,</p>
<p>When the workflow is triggered by the events related to issue (<a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#issues" rel="noopener nofollow ugc"><strong>issues</strong></a> or <a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#issue_comment" rel="noopener nofollow ugc"><strong>issue_comment</strong></a>), from the <a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#github-context" rel="noopener nofollow ugc">github context</a> of the workflow run you can get the details of the issue via the object “<strong>github.event.issue</strong>”.<br>
For example:</p>
<ul>
<li>
<strong>github.event.issue.title</strong> - - the <strong>title</strong> of the issue.</li>
<li>
<strong>github.event.issue.body</strong> - - the <strong>body</strong> (or <strong>description</strong>) of the issue, it is the first comment when creating the issue.</li>
</ul>
<p>A simple workflow example:</p>
<pre><code class="lang-yaml">on: issues

jobs:
  job1:
    runs-on: ubuntu-latest
    steps:
      - name: View issue information
        run: |
          echo "Issue title: ${{ github.event.issue.title }}"
          echo "Issue body: ${{ github.event.issue.body }}"

# You also can print the whole github context to the logs to view more details.
      - name: View the github context
        run: echo "$GITHUB_CONTEXT"
        env:
          GITHUB_CONTEXT: ${{ toJson(github) }}
</code></pre>

**Comment by abineyard** [comment]
Is there a way to go deeper than just body? Like if you had a dropdown list of stores (id: store) and you want to just display the store names, could you do something like:
"${{ github.event.issue.body.store }}"

**Reply by thechelsuk** [reply]
I had tried this at the time, and there wasn’t anyway to access this level of data.
